packages/flutter/lib/src/widgets/nested_scroll_view.dart

```diff
@@ -1384,6 +1441,32 @@ class _NestedScrollPosition extends ScrollPosition implements ScrollActivityDele
     return 0.0;
   }
 
+
+  // Returns the amount of delta that was not used.
+  //
+  // Negative delta means going down (exposing stuff above), positive delta
+  // going up (exposing stuff below).
```

Comment:
This is a bit confusing, since the AxisDirection and GrowthDirection are also part of determining 'up' and 'down'. We can simplify this by saying a negative delta represents a forward ScrollDirection, while the positive would be a reverse ScrollDirection.

Reply:
Thank you for the suggestion.